projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -21,6 +21,57 @@ describe('Profile-tag component', () => {
 
     profileTagHelper.triggerLoaded();
   });
+  it('should send a CartChanged event on adding an item to cart', () => {
+    goToProductPage();
+    cy.get('cx-add-to-cart button.btn-primary').click();
+    cy.get('cx-added-to-cart-dialog .btn-primary');
+    cy.window().then(win => {
+      expect((<any>win).Y_TRACKING.eventLayer.length).to.equal(2);
+      expect((<any>win).Y_TRACKING.eventLayer[1]['name']).to.equal(
+        'CartSnapshot'
+      );
+      const cartPayload = JSON.stringify(
+        (<any>win).Y_TRACKING.eventLayer[1]['data']
+      );
+      expect(cartPayload).to.include('cart');
+      expect(cartPayload).to.include('code');
+    });
+  });
+  it('should send an additional CartChanged event on modifying the cart', () => {
+    goToProductPage();
+    cy.get('cx-add-to-cart button.btn-primary').click();
+    cy.get('cx-added-to-cart-dialog .btn-primary').click();
+    cy.get('cx-cart-item-list')
```

Comment:
change this that clicks the '+' to add an item

to 

    cy.get('cx-cart-item cx-item-counter')
      .getByText('+')
      .click();

because above keeps failing as the selector cx-action is not found